handler/src/main/java/io/netty/handler/ssl/DefaultOpenSslApplicationProtocolNegotiator.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import java.util.List;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+/**
+ * OpenSSL {@link ApplicationProtocolNegotiator} for ALPN and NPN.
+ */
+public final class DefaultOpenSslApplicationProtocolNegotiator implements OpenSslApplicationProtocolNegotiator {
```

Comment:
> actually it contradicts our convention for the 'default' implementation classes.

It seems like this package uses a different convention (see https://github.com/netty/netty/tree/master/handler/src/main/java/io/netty/handler/ssl).  The classes here first define context (openssl/jdk) before `Default`.  I may have contributed to this issue and I agree elsewhere it is typical to have the `Default...` first, but it seems like being consistent w.r.t to this package means putting `OpenSsl` first?